asciidoc/volume1/use-cases/tf1-ch-c-use-case-stad.adoc

```diff
@@ -52,6 +52,17 @@ NOTE: The 50ms target accuracy is suitable for highly demanding use cases like r
 ====
 ****
 
+.R1521
+[sdpi_requirement#r1521,sdpi_req_level=should]
+****
+The <<term_manufacturer>> of a <<vol1_spec_sdpi_p_actor_somds_participant>> should configure its <<acronym_ts_service>> client to give priority to system clock adjustments that are slewing (over stepping adjustments).
```

Comment:
That one got missed. I've tried to stick with "non-slewing" and "slewing" adjustments because that's the term  mostly commonly used with NTP and it is less confusing than "step" change. 

Also, R1521 bothers me a lot. I think its intent can be better expressed (no offense intended to original author :) ) with the drivers for and references to RFC5905 and changed it too:
``` adoc
.R1521
[sdpi_requirement#r1521,sdpi_req_level=should]
****
The <<term_manufacturer>> of a <<vol1_spec_sdpi_p_actor_somds_participant>> should configure
its <<acronym_ts_service>> client to prioritize smooth, monotonic, changes to the system clock. 

.Notes
[%collapsible]
====
NOTE: <<vol1_spec_sdpi_p_actor_somds_participant>>s using, for example, <<ref_rfc_5905, NTP>> 
to syncronize their device clock with the <<acronym_ts_service>> could satisfy this requirement by 
following the cold and warm startup algoriths and clock discipline algorithms with tuning parameters 
described in <<ref_rfc_5905>>.

NOTE: <<vol1_spec_sdpi_p_actor_somds_participant>>s using other synchronization standards
should similarly strongly favour slewing (adjusting clock frequency) over non-slewing (large changes forward 
or backward in time) adjustments, and supress non-slewing adjustments for a period during initialization. 

====
****
```

Reply:
Also I started a glossary. I didn't know where to put it yet, so for now at the top of the non-slewing use case. Some editorial normalization of terms is still needed to be consistent everywhere &mdash; once we have the main points, I guess 

Also below for reference:

time-reference frame
A device-specific context for measuring and assigning timestamps to events defined by its rate of passage of time (which may vary over time) and alignment to some external temporal standard (e.g., provided by a <<acronym_ts_service>>). Changes to the time-reference frame, such as non-slewing adjustments, can create distinct epochs with different temporal properties.

epoch
A disctinct period of time characterized by a consistent temporal properties; a single time-reference frame.

timestamp
A point in time obtained from a system clock; while a timestamp is obtained within the context of a time-reference frame, timestamps do not have an intrinsic reference to time-reference frame. 

timestamp version
A unique identifier, within the scope of a MDIB sequence, of a time-reference frame epoc.

slewing time adjustment
Adjustments made to a system clock's frequency. Generally so that the time reported by a system clock matches that of a <<acronym_ts_service>> at some point in the future, within the statistical uncertaintity of the synchronization algorithm.

non-slewing time adjustment, abrupt time adjustment
An abrubt change to a system clock's time-reference frame to match the time reported by a system clock with that from a <<acronym_ts_service>>, within the statistical uncertaintity of the synchronization algorithm, as quickly as possible.

smooth time adjustments
A gradual adjustment to the temporal properites of a time-refernece frame, characterised by a continuous and monotonically increasing progression of timestamps without abrupt jumps or disruptions to the passage of time. Generally so that the time reported by a system clock matches that of a <<acronym_ts_service>> at some point in the future, within the statistical uncertaintity of the synchronization algorithm.

clock-discipline algorithm
The algorithm employed by a <<acronym_ts_service>> client to minimize the error between a reference time source. It main include smooth (e.g., slewing) and, in some cases, abrupt (e.g., non-slewing) corrections.